src/routes/(console)/organization-[organization]/billing/planSummary.svelte

```diff
@@ -239,3 +253,10 @@
         </svelte:fragment>
     </CardGrid>
 {/if}
+
+<style>
+    .loader-container {
+        flex-grow: 1;
+        align-items: center;
```

Comment:
I think we typically use `u-cross-center` for this.

Reply:
Same as other class comment!